rlbot_gui/bot_management/downloader.py

```diff
@@ -43,3 +46,19 @@ def download_gitlfs(repo_url: str, checkout_folder: Path, branch_name: str):
     file.close()
 
     print('Done downloading git repo.')
+
+def download_botfs(repo_url: str, checkout_folder: Path, branch_name: str, bot_path: str):
```

Comment:
what is fs?

can we add a comment about what this method does or change the method name to be a little more readable?

Reply:
file system, just as used in the already existing function download_gitlfs()